File: samples/python/quickstart/responses/quickstart-responses.py
from azure.identity import DefaultAzureCredential
from azure.ai.projects import AIProjectClient

# Format: "https://resource_name.ai.azure.com/api/projects/project_name"
FOUNDRY_PROJECT_ENDPOINT = "your_project_endpoint"
FOUNDRY_MODEL_NAME = "gpt-5-mini"  # supports all Foundry direct models

# Create project and openai clients to call Foundry API
project = AIProjectClient(
    endpoint=FOUNDRY_PROJECT_ENDPOINT,
    credential=DefaultAzureCredential(),
)
openai = project.get_openai_client()

# Run a responses API call
response = openai.responses.create(
    model=FOUNDRY_MODEL_NAME,
    input="What is the size of France in square miles?",
)
print(f"Response output: {response.output_text}")

File: samples/typescript/quickstart/responses/src/quickstart-responses.ts
import { DefaultAzureCredential } from "@azure/identity";
import { AIProjectClient } from "@azure/ai-projects";

// Format: "https://resource_name.ai.azure.com/api/projects/project_name"
const FOUNDRY_PROJECT_ENDPOINT = "your_project_endpoint";
const FOUNDRY_MODEL_NAME = "gpt-5-mini";  // supports all Foundry direct models

async function main(): Promise<void> {
    // Create project and openai clients to call Foundry API
    const project = new AIProjectClient(FOUNDRY_PROJECT_ENDPOINT, new DefaultAzureCredential());
    const openai = await project.getOpenAIClient();

    // Run a responses API call
    const response = await openai.responses.create({
        model: FOUNDRY_MODEL_NAME,
        input: "What is the size of France in square miles?",
    });
    console.log(`Response output: ${response.output_text}`);
}

main().catch(console.error);